Default Namespace labels (#26995)
kubernetes/website: content/en/docs/concepts/services-networking/network-policies.md

supports the `endPort` field in NetworkPolicy specifications.
{{< /note >}}

## Targeting a Namespace by its name

> As of Kubernetes v1.21, a default immutable label ``kubernetes.io/metadata.name`` with the namespace name will be added to all namespaces. While NetworkPolicy cannot target a namespace by its name with some object field, this label now can be used to target a specific namespace.

Reviewer: ```suggestion
As of Kubernetes v1.21, an immutable label `kubernetes.io/metadata.name` is added to all
namespaces. The value of the label is the namespace name. While NetworkPolicy cannot
target a namespace by its name with some object field, this label now can be used to target
a specific namespace.
```
